app/src/main/java/org/fossasia/openevent/app/login/LoginActivity.java

```diff
@@ -57,11 +68,30 @@ protected void onCreate(Bundle savedInstanceState) {
         // Notify presenter to attach
         presenter.attach(this);
 
+        checkBoxEnableUrl.setOnCheckedChangeListener((buttonView, isChecked) -> {
+            if(isChecked) {
+                checkBoxEnableUrl.setTextColor(Color.parseColor("#000000"));
+                addUrlContainer.setVisibility(View.GONE);
+            }
+            else {
+                checkBoxEnableUrl.setTextColor(Color.parseColor("#808080"));
+                addUrlContainer.setVisibility(View.VISIBLE);
+            }
+        });
+
         btnLogin.setOnClickListener(v -> {
 
             String email = etEmail.getText().toString();
             String password = etPassword.getText().toString();
 
+            if(checkBoxEnableUrl.isChecked()) {
```

Comment:
@iamareebjamal here we are setting `Const.BASE_URL`. This can be shifted to the presenter.